buildone

```diff
@@ -41,54 +41,70 @@ log() {
     echo "$@" >&2
 }
 
+is_snapshot() {
+    local -r dist_snapshot_regex="^(jessie|stretch|buster|unstable)-snapshot"
+    if [[ $1 =~ $dist_snapshot_regex ]]; then
+        true
+    else
+        false
+    fi
+}
```

Comment:
I always find these constructs funny:
```c
if (foo) {
  return true;
} else {
  return false;
}
```

turns out they are unnecessary even in bash:

```suggestion
is_snapshot() {
    local -r dist_snapshot_regex="^(jessie|stretch|buster|unstable)-snapshot"
    [[ $1 =~ $dist_snapshot_regex ]]
}
```